entity_token.info

```diff
@@ -4,4 +4,4 @@ backdrop = 1.x
 type = module
 
 dependencies[] = entity
-dependencies[] = entity_plus
+dependencies[] = entity_plus (>1.x-1.0.13)
```

Comment:
Try `dependencies[] = entity_plus (>=1.0.13)`

Reply:
@laryn, I've tried the pattern you mentioned here (except that I used `>=1.0.14` since we want to require the next version of Entity Plus - the current version is 1.0.13, which doesn't contain the fix). I'm testing in my local environment, adding that line to `entity_token.info`. I am still able to enable Entity Tokens, despite the fact that I have Entity Plus version 1.0.13 installed in my environment. I cleared caches. The fact that it's still letting me enable Entity Tokens is telling me that this check is not working, unless I'm missing something. Any thoughts will be welcome. 

This may not a big deal, though. I have also tested patching Entity Plus with the new code for date metadata, AND leaving Entity Tokens as is (keeping the code I deleted in this PR). There are no function clashes since I actually renamed the functions in Entity Plus. So, the patch still "works" even if someone were to update Entity Plus, but not Entity Tokens by mistake.

Reply:
But of course the issue is if someone were to update Entity Tokens and not Entity Plus - they end up with wrapper access to date fields...

Reply:
@laryn, I've found a bug in the dependencies check in core and have reported it. Backdrop fails to correctly check for versions that use 3 digits (e.g. 1.0.13) instead of 2.  (as in 1.2 , etc). See [backdrop/backdrop-issues#/5356 ](https://github.com/backdrop/backdrop-issues/issues/5356)